src/search.h

```diff
@@ -60,6 +60,8 @@ typedef struct GeanySearchPrefs
 	gboolean	use_current_file_dir;	/* find in files directory to use on showing dialog */
 	gboolean	hide_find_dialog;		/* hide the find dialog on next or previous */
 	gboolean	replace_and_find_by_default;	/* enter in replace window performs Replace & Find instead of Replace */
+	gboolean	skip_confirmation_for_replace_in_session;	/* do *not* ask for confirmation
```

Comment:
Not last == ABI break as well as API

And if its moved to the end its technically not an API break either since it isn't documented for plugin use.

Reply:
Thanks, fixed.

Isn't it part of the API by https://github.com/geany/geany/blob/master/src/plugindata.h#L202?

Reply:
The struct is, but your new field is not doxygened so it won't appear in the API docs, in fact only `use_current_word` appears in the [API docs](https://www.geany.org/manual/reference/structGeanySearchPrefs.html) from the whole structure.  If you want it in the API docs you need to use a `/**<` comment on the field but unless you are about to use it in a plugin its probably better to not doxygen it and no change needed in the API version.

Reply:
Ok, I reverted the API version bump.

I was referring to that the new field is technically part of the plugin API as I can access and use it from a plugin, no matter whether it is doxgened.
Anyway, using the documented API as source is also fine.